client/src/components/settings/toggle-setting.tsx

```diff
@@ -31,7 +31,7 @@ export default function ToggleSetting({
   ...restProps
 }: ToggleSettingProps): JSX.Element {
   return (
-    <Fragment>
+    <>
```

Comment:
Same here.

Reply:
There is **ButtonSpacer** component immediately after the dev tag , so <> cannot be ommited...

Reply:
Ah! Sorry, I missed that.